cmd/containerd-shim-runhcs-v1/service.go

```diff
@@ -42,10 +64,35 @@ type service struct {
 	taskOrPod atomic.Value
 
 	// cl is the create lock. Since each shim MUST only track a single task or
-	// POD. `cl` is used to create the task or POD sandbox. It SHOULD not be
+	// POD. `cl` is used to create the task or POD sandbox. It SHOULD NOT be
 	// taken when creating tasks in a POD sandbox as they can happen
 	// concurrently.
 	cl sync.Mutex
+
+	// shut is closed to signal a shut request is received
+	shut chan struct{}
```

Comment:
shut (to me at least haha) is an awkward name, it doesn't jump out to me as an abbreviation for shutdown

Reply:
I was looking for other abbreviations that didn't use `sht`, which felt ... easily misconstrued. 
I changed it to `shutdown`